controlplane/pkg/prefix_pool/prefixpool_test.go

```diff
@@ -64,6 +64,23 @@ func TestNetExtract1(t *testing.T) {
 	Expect(err).To(BeNil())
 }
 
+func TestNetExtract2(t *testing.T) {
```

Comment:
Please rename these to `TestNetExtractIPv4` and `TestNetExtractIPv6`.

Reply:
Resolved 👍